scripts/consensus_merge.py

```diff
@@ -0,0 +1,587 @@
+                open in 'r' mode
+            hotspot (bool): whether this variant is in a mutational hotspot
+                (default False)
+    """
+    variant_lookup = {v.caller: v for v in single_caller_variants}
+
+    output_record = output_vcf.new_record()
+    # Set consistent attributes
+    liftover_record = single_caller_variants[0].record
+    output_record.chrom = liftover_record.chrom
+    output_record.ref = liftover_record.ref
+    output_record.alts = liftover_record.alts
+    output_record.id = liftover_record.id
+    output_record.start = liftover_record.start
+    output_record.stop = liftover_record.stop
+
+    # For each caller, get information required for format fields for this variant
+    joint_tags = ('GT', 'AD', 'AF', 'DP')
+    no_call_info = {tag: '.' for tag in joint_tags} 
+    allele_information = {caller.lower(): {} for caller in CALLER_NAMES}
+
+    for variant in single_caller_variants:
+        normal_sample, tumor_sample = variant.record.samples
+        variant.normal = Sample(normal_sample, 'normal', variant.record, variant.caller)
+        variant.tumor = Sample(tumor_sample, 'tumor', variant.record, variant.caller)
+
+    for index in (0, 1):
+        normal = 0
+        tumor = 1
+        GT_list = []
+        AD_list = []
+        AF_list = []
+        DP_list = []
+        for caller in CALLER_NAMES:
+            try:
+                targ_var = variant_lookup[caller.lower()]
+            except KeyError:
+                for flist in (GT_list, AD_list, AF_list, DP_list):
+                    flist.append('.')
+                continue
+            if index == normal:
+                targ_sample = targ_var.normal
+            elif index == tumor:
+                targ_sample = targ_var.tumor
+                
+            GT_list.append(str(targ_sample.GT))
+            AD_list.append(stringify(targ_sample.AD))
+            AF_list.append('{:0.4f}'.format(targ_sample.AF))
+            DP_list.append(str(targ_sample.DP))
+
+        GT = stringify(GT_list, FORMAT_JOIN)
+        AD = stringify(AD_list, FORMAT_JOIN)
+        AF = stringify(AF_list, FORMAT_JOIN)
+        DP = stringify(DP_list, FORMAT_JOIN)
+
+        output_record.samples[index]['GTC'] = GT
+        output_record.samples[index]['ADC'] = AD
+        output_record.samples[index]['AFC'] = AF
+        output_record.samples[index]['DPC'] = DP
+
+        consensus_gt, gt_tag = get_gt_consensus(GT_list)
+
+        output_record.samples[index]['AGT'] = consensus_gt
+        output_record.samples[index]['GT_STATUS'] = gt_tag
+    
+    chrom = single_caller_variants[0].record.chrom
+    pos = single_caller_variants[0].record.pos
+    mapq, mq0 = get_mapq(normal_cram, chrom, pos)
+
+    output_record.info['MQ'] = mapq
+    output_record.info['MQ0'] = mq0
+    output_record.info['CAL'] = ','.join([c for c in CALLER_NAMES if c.lower() in variant_lookup])
+
+    if hotspot:
```

Comment:
So this is actually going to end up in the `INFO` field, not format. Split multiallelics is able to properly assign which variant it should go to as it will be a csv field, a `0` and `1` per allele.

Reply:
Ok, I switched this to INFO and all the filters will just be PASS, although can also just use '.'